target_chains/ethereum/sdk/solidity/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@pythnetwork/pyth-sdk-solidity",
+  "version": "2.2.0",
+  "description": "Read prices from the Pyth oracle",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pyth-network/pyth-crosschain",
+    "directory": "target_chains/ethereum/sdk/solidity"
+  },
+  "scripts": {
+    "format": "npx prettier --write .",
+    "generate-abi": "node scripts/generateAbi.js",
+    "build": "solcjs --bin MockPyth.sol -o build/"
```

Comment:
this used to be named "build mock", and i changed it to build so it gets built with all the other packages above.